db.json

```diff
@@ -1066,7 +1117,8 @@
     "extensions": ["pgp"]
   },
   "application/pgp-keys": {
-    "source": "iana"
+    "source": "iana",
+    "extensions": ["key"]
```

Comment:
Where does this come from? The official record for this type (https://tools.ietf.org/html/rfc3156#section-9.3) states the file extension is ".asc". It jumped out to me since ".key" is the Apple Keynote files.

Reply:
gnupg uses this for keys.

Reply:
Gotcha. Can you link to the source of that information?